### PR TITLE
Sleet - more applepay request logic

### PR DESCRIPTION
**Pull request by nirajjayantbolt**

### CODE REVIEW

File: gateways/orbital/request_builders_test.go [hunk 1]
```diff
@@ -120,6 +124,37 @@ func TestBuildAuthRequest(t *testing.T) {
...
+				Body: RequestBody{
+					OrbitalConnectionUsername: "username",
+					OrbitalConnectionPassword: "password",
+					MerchantID:                1,
+					IndustryType:              IndustryTypeEcomm,
+					MessageType:               MessageTypeAuth,
+					BIN:                       BINStratus,
+					TerminalID:                TerminalIDStratus,
+					XMLName:                   xml.Name{Local: RequestTypeNewOrder},
+					AccountNum:                mastercardBase.CreditCard.Number,
```
Comment (dbook13) [inline gateways/orbital/request_builders_test.go:140]: should we be checking applepayBase here?

File: gateways/orbital/request_builders_test.go [hunk 2]
```diff
@@ -120,6 +124,37 @@ func TestBuildAuthRequest(t *testing.T) {
...
+					Exp:                       "202010",
+					CardSecVal:                mastercardBase.CreditCard.CVV,
```
Comment (dbook13) [inline gateways/orbital/request_builders_test.go:142]: same as above

File: gateways/orbital/request_builders_test.go [hunk 3]
```diff
@@ -120,6 +124,37 @@ func TestBuildAuthRequest(t *testing.T) {
...
+					CurrencyCode:              CurrencyCodeUSD,
+					CurrencyExponent:          CurrencyExponentDefault,
+					Amount:                    100,
+					OrderID:                   *mastercardBase.ClientTransactionReference,
+					AVSzip:                    *mastercardBase.BillingAddress.PostalCode,
+					AVSaddress1:               *mastercardBase.BillingAddress.StreetAddress1,
+					AVSaddress2:               mastercardBase.BillingAddress.StreetAddress2,
+					AVSstate:                  *mastercardBase.BillingAddress.RegionCode,
+					AVScity:                   *mastercardBase.BillingAddress.Locality,
+					AVScountryCode:            *mastercardBase.BillingAddress.CountryCode,
```
Comment (dbook13) [inline gateways/orbital/request_builders_test.go:152]: same as above

File: gateways/orbital/request_builders_test.go (hunk 1 above)
Comment (nirajjayantbolt) [reply gateways/orbital/request_builders_test.go:140]: oops yeah I guess it doesnt really matter since w edont change anything but good for consistency

**Pull request merged**